app/src/main/java/org/kiwix/kiwixmobile/di/modules/ApplicationModule.java

```diff
@@ -30,4 +31,8 @@ public ApplicationModule(KiwixApplication application) {
   BookUtils provideBookUtils() {
     return new BookUtils();
   }
+
```

Comment:
Prefer injection through constructor rather than using `@Provides`. But you should also add the annotation to the class I have described in the other comment.

Reply:
Yes, Thanks. I was using ```@Provides``` initially and then shifted to Constructor injection but forgot to remove the redundant method.